sound/soc/sof/sof-audio.h

```diff
@@ -290,6 +290,9 @@ struct snd_sof_pcm_stream {
 	 * active or not while suspending the stream
 	 */
 	bool suspend_ignored;
+
+	/* calculate delay for different ipc version */
```

Comment:
the comment actually doesnt make much sense. This is a pointer to IPC-specific data structure that is used for timestamping